### PR TITLE
Css adjustement

### PR DESCRIPTION
**Pull request by BBriset**

### CODE REVIEW

**Review by BenZen59 — DISMISSED** [review]

File: src/components/Categories/Categories.jsx [hunk 1]
```diff
@@ -32,7 +42,14 @@ export default function Categories() {
 
   return (
     <div>
-      <div>
+      <div className="head">
+        <h1 className="choose">
+          {' '}
```
Comment (Hacene010) [inline src/components/Categories/Categories.jsx:47]: Remove this

File: src/components/Categories/Categories.css [hunk 2]
```diff
@@ -1,82 +1,146 @@
...
+    height: 70px;
+    overflow: auto;
+    border: 1px solid #f76c6c;
+  }
+
+  .categories:hover {
+    transform: translateY(0);
+  }
+
+  .movieimg {
```
Comment (Hacene010) [inline src/components/Categories/Categories.css]: camelCase

File: src/components/filterButton/FilterButton.jsx [hunk 3]
```diff
@@ -3,7 +3,11 @@ import './FilterButton.css';
 
 function FilterButton() {
   return (
-    <Link to="Categories/" className="filterButtonBlock">
+    <Link
+      to="Categories/"
```
Comment (Hacene010) [inline src/components/filterButton/FilterButton.jsx]: Are you sure this will work ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
There are some evils

**Review by BenZen59 — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**